packages/e2e/test/long-running/multisig-wallet/multisig-delegation-rewards.test.ts

```diff
@@ -0,0 +1,312 @@
+    const { keyAgent: charlotteKeyAgent, pubKey: charlottePubKey } = await getKeyAgent(
+      charlotteMnemonics,
+      faucetWallet
+    );
+
+    const multiSigParticipants = [alicePubKey, bobPubKey, charlottePubKey];
+
+    const aliceMultiSigWallet = await createMultiSignWallet(aliceKeyAgent, faucetWallet, multiSigParticipants);
+    const bobMultiSigWallet = await createMultiSignWallet(bobKeyAgent, faucetWallet, multiSigParticipants);
+    const charlotteMultiSigWallet = await createMultiSignWallet(charlotteKeyAgent, faucetWallet, multiSigParticipants);
+
+    expect(aliceMultiSigWallet.getPaymentAddress()).toEqual(bobMultiSigWallet.getPaymentAddress());
+    expect(aliceMultiSigWallet.getPaymentAddress()).toEqual(charlotteMultiSigWallet.getPaymentAddress());
+
+    await fundMultiSigWallet(faucetWallet, bobMultiSigWallet.getPaymentAddress());
+
+    const multiSigWalletBalance = await firstValueFrom(
+      bobMultiSigWallet.getBalance().pipe(
+        map((value) => value.coins),
+        filter((value) => value > 0n),
+        take(1)
+      )
+    );
+
+    expect(multiSigWalletBalance).toBeGreaterThan(0n);
+
+    const [poolId] = await getPoolIds(faucetWallet.stakePoolProvider, 1);
+
+    // Alice will initiate the delegation transaction on her wallet.
+    let tx = await aliceMultiSigWallet.delegate(poolId);
+    tx = await aliceMultiSigWallet.sign(tx);
+    tx = await bobMultiSigWallet.sign(tx);
+    tx = await charlotteMultiSigWallet.sign(tx);
+
+    const txId = await charlotteMultiSigWallet.submit(tx);
+
+    // Search chain history to see if the transaction is there.
+    const txFoundInHistory = await firstValueFrom(
+      charlotteMultiSigWallet.getTransactionHistory().pipe(
+        map((txs) => txs.find((hTx) => hTx.id === txId)),
+        filter(isNotNil),
+        take(1)
+      )
+    );
+
+    expect(txFoundInHistory).toBeTruthy();
+
+    // Delegation is completed. Now we wait for rewards to be available.
+    const delegationTxConfirmedAtEpoch = await getTxConfirmationEpoch(
+      charlotteMultiSigWallet.getTransactionHistory(),
+      tx.getTransaction(),
+      faucetWallet.eraSummaries$
+    );
+
+    logger.info(`Delegation tx confirmed at epoch #${delegationTxConfirmedAtEpoch}`);
+
+    await waitForEpoch(faucetWallet, delegationTxConfirmedAtEpoch + 2);
+    await generateTxs(faucetWallet, faucetWallet);
+    await waitForEpoch(faucetWallet, delegationTxConfirmedAtEpoch + 4);
+
+    // Check reward
+    const multiSigWalletRewardBalance = await firstValueFrom(
+      bobMultiSigWallet.getRewardAccountBalance().pipe(
+        filter((value) => value > 0n),
+        take(1)
+      )
+    );
+
+    expect(multiSigWalletRewardBalance).toBeGreaterThan(0n);
+
+    logger.info(`Generated rewards: ${multiSigWalletRewardBalance} tLovelace`);
+
+    tx = await aliceMultiSigWallet.transferFunds(faucetAddress, { coins: 2_000_000n });
```

Comment:
This is an important part of the test, best to be explicit about it
```suggestion
    tx = await aliceMultiSigWallet.transferFunds(faucetAddress, { coins: 2_000_000n });
    expect(tx.withdrawals.length).toBeGreaterThan(0);
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/992/commits/8664427d551aebded9b0979446284bf483dc3b91